grails-app/assets/javascripts/managementUnitReport.js

```diff
@@ -0,0 +1,36 @@
+function ManagementUnitReportSelectorViewModel() {
+    var self = this;
+    var now = convertToSimpleDate(new Date());
+    var d = new Date();
+    d.setFullYear(2018, 6, 1);
+    self.fromDate = ko.observable(d).extend({simpleDate:false});
+    self.toDate = ko.observable(now).extend({simpleDate:false});
+
+    self.muReportDownloadSummary = function() {
+        var summaryFlag = true;
+        generateMuReport(summaryFlag);
+    };
+
+    self.muReportDownload = function () {
+        generateMuReport();
+    };
+
+    function generateMuReport(summaryFlag = false) {
+        var selectPeriod = $('select#reportPeriodOfManagementUnit').val()
+        var fromDate = $('#fromDate').val()
```

Comment:
Don't use selectors to get the value inside a view model.  The value is already bound to self.fromDate and self.toDate.